package.json

```diff
@@ -14,10 +14,20 @@
   "author": "Andreas Hocevar",
   "license": "BSD-2-Clause",
   "scripts": {
+    "karma": "karma start test/karma.conf.js",
```

Comment:
What about also adding a `"test"` script? Then we can set up a CI too.

Reply:
yes, good idea, done